src/EnrollDuoController.js

```diff
@@ -10,15 +10,15 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/*jshint camelcase:false, newcap:false */
+/* eslint camelcase: 0*/
 define([
   'okta',
   'duo',
   'vendor/lib/q',
   'util/FormController',
   'views/enroll-factors/Footer'
 ],
-function (Okta, Duo, Q, FormController, Footer) {
+function (Okta, Duo, q, FormController, Footer) {
```

Comment:
Can we disable this rule (i.e. with jshint it was the newcap flag)? Ideally we would reference Q the same way as in the q docs.